spec/support/expire_cookies.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+module ExpireCookies
+  def expire_cookies
```

Comment:
I try to simulate closing broweser here, and if I 'wait' with time cop cookies which have expire date expires (and I don't need it), but my session cookie, for example, has not expire date and will be alive (and thats a problem). It's not about time it's about how browser interact with cookies when it closing.